PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebVaultRequest.kt

```diff
@@ -6,7 +6,16 @@ package com.paypal.android.paypalwebpayments
  * @property [setupTokenId] ID for the setup token associated with the vault approval
  * @property [approveVaultHref] URL for the approval web page
  */
-data class PayPalWebVaultRequest(
+data class PayPalWebVaultRequest @Deprecated("Use PayPalWebVaultRequest(setupTokenId:) instead.")
+constructor(
     val setupTokenId: String,
-    val approveVaultHref: String
-)
+    val approveVaultHref: String?? // NEXT_MAJOR_VERSION: - Remove this property
```

Comment:
```suggestion
    val approveVaultHref: String? // NEXT_MAJOR_VERSION: - Remove this property
```

Reply:
We may be able to mark the property as deprecated as well. I'm not sure if that's useful to merchants, but it couldn't hurt if it does work.